### ISSUE TITLE
AnyChart Trial Version watermark

### ISSUE THREAD

**Issue by AbanoubWagdyy** [AnyChart/AnyChart-Android#119]
AnyChart Trial Version under each graph can not be hidden 

I am using the following gradle dependency : 
implementation 'com.github.AnyChart:AnyChart-Android:1.1.2'

how can i hide this watermark value ?
Thanks

**Comment by Shestac92** [comment]
@AbanoubWagdyy 
[The wiki article](https://github.com/AnyChart/AnyChart-Android/wiki/Credits) describes how to adjust or disable chart credits. The purchased license is required to disable credits.

**Comment by johnnyzen** [comment]
Your business model is not the best. Adding watermarks and not being clear about it until I have added the library is not the way for me to buy the license.

All you have done is annoyed me to the point that I now dont want to use the library at all.

**Comment by Shestac92** [comment]
@johnnyzen 
We are very sorry that this caused you any inconvenience. If you have any questions or suggestions about terms of using the library, please, contact us _sales@anychart.com_ or _support@anychart.com_

**Comment by hp1227** [comment]
@Shestac92 
I want to remove the trial watermarks.
I used several kinds of chart in the one project.
![image](https://user-images.githubusercontent.com/47973689/65898352-f2acc100-e3e3-11e9-8c7a-2c0350eeb095.png)
If i check out this item, do the watermarks disapper?

**Comment by Shestac92** [comment]
@hp1227 
Any type of purchased license allows disabling trial credits.
Please, contact _sales@anychart.com_ to probe about the required license type for your project/application.

**Comment by topherbuckley** [comment]
I agree with @johnnyzen . I would have considered purchasing a license had it been clear in the README, but now I invested some time to read your tutorials and API a bit, only to find it is not free to use. Please update your licensing section of the README to inform users that this is a paid-to-use library, but free to try with watermarks enabled.

**Comment by johnnyzen** [comment]
3 years after my comment and still the same!!!!. I just checked the read me and still no mention of trial. I went on to use mpchart. Come on guys. Just put something in the readme!!!!

**Comment by topherbuckley** [comment]
@johnnyzen  thanks for linking me to mpchart. I was trying out [androidplot](https://github.com/halfhp/androidplot) but [mpchart](https://github.com/PhilJay/MPAndroidChart) looks a bit further along also with a CLEARLY labeled Apache 2.0 license in the README. I'd feel bad about advertising other projects here, but github literally creates a popup when you create a repo saying you should attach a license to your code...Its github 101 ...add a readme, add a license, add a .gitignore... develop.

**Comment by johnnyzen** [comment]
@topherbuckley you will have great joy with mpchart. (sorry AnyChart)
